### PR TITLE
Replace usages of YUI framework by simple JS/CSS

### PR DESCRIPTION
**Pull request by zbynek**
This replaces a few usages of YUI in core:
* session storage for sortable tables (no need for compatibility layer provided by YUI)
* repeateble form elements use standard CSS transition instead of YUI animations
* avoid JS function `makeButton` and CSS selectors that trigger it (`submit-button`, `yui-button`)

This should mostly not cause changes in the UI except for a few button styles
<details>
<summary>Screenshots</summary>

Rekey dialog before:

rekey-before

After:

rekey-after

Expandable console note (after)

console-light
console-dark

Slow trigger monitor (after)

monitor-dark

</details>

### Testing done

* Manually tested the buttons in screenshot (had to hack the Java code to create custom Expandable Note and trigger the monitors)
* Manually tested adding steps to freestyle build and columns to a view to check the animations
* Manually checked that table sorting is preserved on refresh in dashboard

There are no new tests as there is no new functionality here.

### Proposed changelog entries

- Remove some usages of YUI framework in the user interface.

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [ ] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [x] There is automated testing or an explanation as to why this change has no tests.
- [x] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [x] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [x] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [x] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [x] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@mention

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

**Review by timja — APPROVED** [review]
Thanks!

**Review by janfaracik — APPROVED** [review]

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
